utils/github_api.py

```diff
@@ -0,0 +1,33 @@
+import os
+from pathlib import Path
+from urllib.parse import urlparse
+
+import github
+
+from utils import secret_reader
+
+
+GH_BASE_URL = os.environ.get('GITHUB_API', 'https://api.github.com')
+
+
+class GithubApi:
+    def __init__(self, instance, project_url, settings):
```

Comment:
I guess you wanted to make this class similar to `GitLabApi` but `project` in GitHub is something different. I would rename `project_url` to `name_url` as it is a bit misleading